modules/cnv-define-tcp-liveness-probe.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc
+
+[id="cnv-define-tcp-liveness-probe_{context}"]
+
+= Define a TCP liveness probe
+
+This procedure provides an example configuration file for defining
+TCP liveness probes.
+
+.Procedure
+
+. Customize a YAML configuration file to create an TCP liveness probe, using
+this code block as an example. In this example:
+* You configure a probe using `spec.livenessProbe.tcpSocket`, which queries port `1500` of the
+`VirtualMachineInstance`, after an initial delay of `120` seconds.
+* The `VirtualMachineInstance` installs and runs a minimal HTTP server
+on port `1500` using `cloud-init`.
```

Comment:
As with the other module, it might be better to include this info in callouts in the code block, but I think it's also okay as is.

Reply:
I'll keep that in mind for future... sounds like a good idea.